src/authV2/HISTORY.rst

```diff
@@ -5,4 +5,8 @@ Release History
 
 0.1.0
 ++++++
-* Initial release.
\ No newline at end of file
+* Initial release.
+
+0.1.2
+++++++
+* Bug fix for '--unauthenticated-client-action' option in webapp auth, updated allowed values.
```

Comment:
Please add the new history notes to the top of the file

Reply:
Just moved it!